app/models/journeys/further_education_payments/provider/authorisation.rb

```diff
@@ -2,24 +2,22 @@ module Journeys
   module FurtherEducationPayments
     module Provider
       class Authorisation
-        def initialize(answers:, slug:)
+        def initialize(answers:)
```

Comment:
auth no longer needs to know where the user is. it is now more about are they authorised for the whole journey